spec/services/orientations/save_spec.rb

```diff
@@ -94,31 +153,35 @@
           end
 
           it "outputs an error" do
-            expect(subject.errors).to eq(["Les dates se chevauchent avec une autre orientation"])
+            expect(subject.errors.first).to include("Cette orientation chevauche une autre orientation")
           end
         end
 
         context "when it is between two orientations" do
           let!(:starts_at) { Date.parse("10/11/2022") }
 
+          let!(:second_orientation) do
+            create(:orientation, user:, starts_at: Date.parse("21/11/2022"), ends_at: nil)
+          end
+
           it "sets the ends_at at the posterior starts_at" do
             subject
             is_a_success
-            expect(orientation.reload.ends_at).to eq(Date.parse("11/11/2022"))
+            expect(orientation.reload.ends_at).to eq(Date.parse("20/11/2022"))
             expect(second_orientation.reload.ends_at).to be_nil
           end
         end
       end
 
       context "when the time range includes an other orientation" do
-        let!(:starts_at) { Date.parse("09/10/2022") }
+        let!(:starts_at) { Date.parse("10/10/2022") }
 
         it "is a failure" do
           is_a_failure
         end
 
         it "outputs an error" do
-          expect(subject.errors).to eq(["Les dates se chevauchent avec une autre orientation"])
+          expect(subject.errors.first).to include("Cette orientation chevauche")
```

Comment:
pour être homogène ;)
```suggestion
          expect(subject.errors.first).to include("Cette orientation chevauche une autre orientation")
```